yarn.lock

```diff
@@ -1284,9 +1284,13 @@ yargs@16.2.0:
     require-directory "^2.1.1"
     string-width "^4.2.0"
     y18n "^5.0.5"
+<<<<<<< HEAD
```

Comment:
Looks like a merge wasn't done correctly, please remove the text on this line and 1289

Reply:
Resolved. Not sure what git did.